x/profiles/keeper/keeper.go

```diff
@@ -55,6 +68,12 @@ func (k Keeper) IsUserBlocked(ctx sdk.Context, blocker, blocked string) bool {
 // It assumes that the given profile has already been validated.
 // It returns an error if a profile with the same DTag from a different creator already exists
 func (k Keeper) StoreProfile(ctx sdk.Context, profile *types.Profile) error {
+
+	if _, found := k.GetAccountByChainLink(ctx, "desmos", profile.GetAddress().String()); found {
```

Comment:
I think we should define a constant named `DesmosChainName` inside the `types/keys.go` file and use that here instead of manually using `"desmos"`